plugins/events-backend-module-google-pubsub/src/util/createPatternResolver.test.ts

```diff
@@ -40,11 +40,6 @@ describe('createPatternResolver', () => {
         a: { b: ['first', 'second'] },
       }),
     ).toEqual('-second-');
-    expect(
-      createPatternResolver('-{{ a.b.0 }}-')({
```

Comment:
When reusing the new `valueAtPath`, this syntax is no longer supported. But I feel that that's pretty much a good thing - it wasn't really intuitive or needed